### ISSUE TITLE
Command line help of Export Report is not correct

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#3015]
Steps to reproduce the issue:
1. Run ZAP, with Export Report add-on installed, using the command line argument `-help`;
2. Note in the outputted help that some of the arguments are repeated:
```
[...]
Add-on options:
	-last_scan_report <path> Generate the 'Last Scan Report' into the specified path
	-last_scan_report <path> Generate the 'Last Scan Report' into the specified path
	-last_scan_report <path> Generate the 'Last Scan Report' into the specified path
	-last_scan_report <path> Generate the 'Last Scan Report' into the specified path
	-last_scan_report <path> Generate the 'Last Scan Report' into the specified path
	-addoninstall <addon>    Install the specified add-on from the ZAP Marketplace
[...]
```
four of the `-last_scan_report` correspond to `-export_report`, `-source_info`, `-alert_severity` and `-alert_details`, the ones provided by the add-on (which have incorrect help info).

Add-on:
Export Report, versions >= 1.

**Comment by JordanGS** [comment]
https://github.com/zaproxy/zap-extensions/blob/alpha/src/org/zaproxy/zap/extension/exportreport/ExtensionExportReport.java#L429-L432

Update to use custom messages.

**Comment by JordanGS** [comment]
Fix in Merge request [#641](https://github.com/zaproxy/zap-extensions/pull/641)

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#641.
